### PR TITLE
Add prefer-template eslint rule

### PR DESCRIPTION
**Pull request by petesantos**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module.exports = {
+  module.exports = {
```
Comment (zarsky-broad) [inline .eslintrc.js]:
```suggestion
module.exports = {
```

File: src/pages/workspaces/workspace/Data.js [hunk 2]
```diff
@@ -640,7 +639,7 @@ class EntitiesContent extends Component {
 
     const header = _.join('\t', [`entity:${entityKey}_id`, ...attributeNames])
 
-    const entityTsv = _.join('\n', [header, ..._.map(entityToRow, sortedEntities)]) + '\n'
+    const entityTsv = `${_.join('\n', [header, ..._.map(entityToRow, sortedEntities)])}\n`
```
Comment (zarsky-broad) [inline src/pages/workspaces/workspace/Data.js:642]: What about this for these?
```suggestion
    const entityTsv = _.join('\n', [header, ..._.map(entityToRow, sortedEntities), ''])
```

Comment (panentheos) [reply src/pages/workspaces/workspace/Data.js:642]: FWIW that suggestion seems less clear to me.

**Review by panentheos — APPROVED** [review]
Click tested, looks good, see whitespace comment.

File: src/pages/workspaces/workspace/Data.js (hunk 2 above)
Comment (petesantos) [reply src/pages/workspaces/workspace/Data.js:642]: kept this as-is, if there is a strong objection to not changing it let me know

**Pull request merged**